pajbot/models/user.py

```diff
@@ -283,8 +283,8 @@ def jsonify(self):
             "points_rank": self.points_rank,
             "subscriber": self.subscriber,
             "moderator": self.moderator,
-            "time_in_chat_online": self.time_in_chat_online.total_seconds(),
-            "time_in_chat_offline": self.time_in_chat_offline.total_seconds(),
+            "time_in_chat_online": 0 if self.time_in_chat_offline is None else self.time_in_chat_online.total_seconds(),
```

Comment:
This should never be None, in which way did you have problems with this?

This value is initialized in `__init__` and also is set to `NOT NULL` on the SQL side.

Reply:
Hmm might be a error with the database i had in the beginning, that is why I did not fix it for the linter, because i was not sure, it was needed, ill go ahead and remove it :)